### DISCUSSION TITLE
Exchange forecasts in the US

### DISCUSSION THREAD

**Discussion by pierresegonne** [electricitymaps/electricitymaps-contrib#7561] [Q&A]
Hey Electricity Maps' community - we're looking for help!

It would be immensely valuable for the Electricity Maps team to access forecasts for exchanges between balancing authorities in the US. In Europe, such data is available through e.g. the [ENTSO-E transparency platform](https://transparency.entsoe.eu/transmission-domain/r2/scheduledCommercialExchangesDayAhead/show) (called `scheduled commercial exchanges`), but we can't find any equivalent in the US. 

Accessing this data would help us improve the accuracy of the data seen for US zones and reduce the influence of the delayed data.

**We're specifically looking for sources of data that tell us what are the scheduled exchanges between two balancing authorities, following the day ahead auction.** 

Please let us know if you have any lead for such data, or if you have any insights that would help us know why trying to look for that data is a lost cause.

Thank you, the Electricity Maps team

**Comment by ryansholin** [comment]
Researching this a bit, and I haven't found any _central_ source of day-ahead data yet, though it looks like "interchange" is reported hourly in Form EIA-930. It looks to me like that's the same historical data already available in Electricity Maps.

Here's the only source of _decentralized_ day-ahead data I have found so far:

- MISO: [Has a REST API](https://data-exchange.misoenergy.org/api-details#api=load-generation-and-interchange-api&operation=get-v1-day-ahead-date-interchange-net-scheduled) **with day-ahead scheduled interchange data**. More docs are [here](https://help.misoenergy.org/knowledgebase/article/KA-01489/en-us).
